include/wx/osx/webview_webkit.h

```diff
@@ -135,15 +120,22 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewWebKit : public wxWebView
 class WXDLLIMPEXP_WEBVIEW wxWebViewFactoryWebKit : public wxWebViewFactory
 {
 public:
-    virtual wxWebView* Create() override { return new wxWebViewWebKit; }
+    virtual wxWebView* Create() override { return CreateWithConfig(nullptr); }
+    virtual wxWebView* CreateWithConfig(std::shared_ptr<wxWebViewConfiguration> config) override { return new wxWebViewWebKit(config); }
     virtual wxWebView* Create(wxWindow* parent,
                               wxWindowID id,
                               const wxString& url = wxWebViewDefaultURLStr,
                               const wxPoint& pos = wxDefaultPosition,
                               const wxSize& size = wxDefaultSize,
                               long style = 0,
                               const wxString& name = wxASCII_STR(wxWebViewNameStr)) override
-    { return new wxWebViewWebKit(parent, id, url, pos, size, style, name); }
+    {
+        auto webView = Create();
+        if (webView->Create(parent, id, url, pos, size, style, name))
```

Comment:
I find it very confusing to see `Create()` called twice, typically this is an error in wx API (you're supposed to either use a non-default ctor or a default ctor and `Create()`). I wonder if there is still any way of renaming something to avoid calling both functions `Create()` or if it's too late?

Reply:
I agree it's confusing but the factory method is called `Create()` but i could of course use new here instead of calling `Create()` from the factory itself.